src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1649,8 +1650,8 @@ public async Task Http2_PendingSend_SendsReset(bool waitForData)
 
                     await Assert.ThrowsAnyAsync<OperationCanceledException>(async () => await client.SendAsync(request, cts.Token));
 
-                    // Delay for a bit to ensure that the RST_STREAM for the previous request is sent before the next request starts.
-                    await Task.Delay(2000);
+                    // Wait until the RST_STREAM for the previous request is received before the next request starts.
+                    await rstReceived.Task.TimeoutAfter(TimeSpan.FromSeconds(60));
```

Comment:
> 60

Do we have a standard timeout constant defined anywhere that we could use for consistency (and the ability to change all timeouts at once later) instead of hardcoding the 60?

Reply:
I already tried searching for such constant, but with no luck.

Reply:
On the second thought, I found [Http2LoopbackServer.Timeout](https://github.com/dotnet/runtime/blob/188243a1ec2abb19f5d37d8274afa1994e848dd2/src/libraries/Common/tests/System/Net/Http/Http2LoopbackServer.cs#L36), but not sure if it's a right thing to use it here.

Reply:
there is TestConfiguration.PassingTestTimeoutMilliseconds but that may be too long. That generally puts cap on test execution.  I think using loopback timeout would be fine in this case as it does not need to precise, right @alnikola? Maybe we can add something DefaultTimeout to loopback and than use it when appropriate. (like the  Http2LoopbackServer.Timeout) That would allows us to adjust it globally if we ever need to on slow platforms.

Reply:
I will try to find all places where we use an explicit timeout and put there some well-known value (e.g. TestConfiguration.PassingTestTimeoutMilliseconds or Http2LoopbackServer.Timeout). Will create a separate PR for this.